README.md

```diff
@@ -1,2 +1,29 @@
-# openapi-componentizer
-OpenAPI Componentizer (GSoC 2023)
+<div align='center'>
+  <img src='OpenAPI_Specification_Logo_Pantone.png' width='35%' title='OpenAPI Logo'/>
+  <h1>Componentizer</h1>
+</div>
+
+  This is a GSoC 2023 project. Initial project is proposed at [Postman Open Technologies GSoC 2023](https://github.com/postman-open-technologies/gsoc-2023/issues/9)
+  ** Mentoring organization: ** [Postman](https://www.postman.com)
+  ** Organizer: ** [Postman Open Technologies](https://github.com/postman-open-technologies)
+  ** Mentors: ** [Bala Avulapati](https://github.com/bavulapati), [Darshan Sen](https://github.com/RaisinTen), [Arvind Kalra](https://github.com/arvindkalra08)
+  ** OpenAPI Specification Lead: ** [Mike Ralphson](https://github.com/MikeRalphson)
```

Comment:
These ones don't look right on the GitHub UI Files Changed tab (everything appears to be on the same line). That needs to be fixed before landing. Will take a look at the rest tomorrow.

Reply:
Nice catch! I didn't notice this earlier. Made the appropriate changes. PTAL